permits/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class BadPermitRequestStatus(Exception):
```

Comment:
Cool. Where is the content of the error defined so that it's understandable by the user ?

Reply:
The user should never see this exception, since it's caught in `redirect_bad_status_to_readonly` and redirects the user to the readonly view of the permit request. I added a more comprehensible `__str__` anyway, to help developers.